label_studio/projects/serializers.py

```diff
@@ -82,6 +98,46 @@ def user_id(self):
     def get_config_has_control_tags(project):
         return len(project.get_parsed_config()) > 0
 
+    @staticmethod
+    def get_config_suitable_for_bulk_annotation(project):
+        li = LabelInterface(project.label_config)
+
+        # List of tags that should not be present
+        disallowed_tags = [
+            LabelsTag,
+            EllipseLabelsTag,
+            KeyPointLabelsTag,
+            PolygonLabelsTag,
+            RectangleLabelsTag,
+            HyperTextLabelsTag,
+            ParagraphLabelsTag,
+            TimeSeriesLabelsTag,
+        ]
+
+        # Return False if any disallowed tag is present
+        for tag_class in disallowed_tags:
+            if li.find_tags_by_class(tag_class):
+                return False
+
+        # For all ChoicesTag and TaxonomyTag, check that perRegion and perItem are not enabled
```

Comment:
I think, we should also check that `Taxonomy` doesn't have `apiUrl` param

Reply:
And most probably `value` on Taxonomy/Choices is "no" as well

Reply:
> And most probably value in Taxonomy/Choices is "no" as well

@Gondragos  I didn't understand this. 
We don't have value in Taxonomy tag for our default config example. Does it mean, that we don't support bulk annotation for it? 

```
<View>
  <Text name="text" value="$text"/>
  <Taxonomy name="taxonomy" toName="text">
    <Choice value="Archaea" />
    <Choice value="Bacteria" />
    <Choice value="Eukarya">
      <Choice value="Human" />
      <Choice value="Oppossum" />
      <Choice value="Extraterrestial" />
    </Choice>
  </Taxonomy>
</View>
```